scripts/mkimg.base.sh

```diff
@@ -256,6 +256,16 @@ create_image_iso() {
 		grub-mkrescue --output ${ISO} ${DESTDIR} -follow-links \
 			-sysid LINUX \
 			-volid "alpine-$PROFILE $RELEASE $ARCH"
+	elif [ "$ARCH" = "s390x" ]; then
+		for _image in vmlinuz initramfs modloop; do
+			cp ${DESTDIR}/boot/$_image-$kernel_flavors ${OUTDIR}
+		done
+		# we ship an example of the parm file
+		printf %s "dasd=0.0.00a0,0.0.00a1 s390x_net=qeth_l2,0.0.00b0,0.0.00b1,0.0.00b2 " > ${OUTDIR}/parmfile
+		printf %s "ip=192.168.0.2::192.168.0.1:255.255.255.0:none:eth0:none:8.8.8.8 " >> ${OUTDIR}/parmfile
+		printf %s "alpine_repo=http://dl-cdn.alpinelinux.org/alpine/latest-stable/main " >> ${OUTDIR}/parmfile
+		printf %s "modloop=http://dl-cdn.alpinelinux.org/alpine/latest-stable/releases/s390x/modloop-$kernel_flavors " >> ${OUTDIR}/parmfile
+		printf %s "ssh_pass=secret " >> ${OUTDIR}/parmfile
```

Comment:
what and where is the parmfile used?

Do we need to install boot loader on the .iso image?

Reply:
The parmfile is used as the configuration file to load into the z/VM hypervisor, along with the kernel and initramfs.

The .iso image is not used for the z/VM hypervisor.

The .iso image is used for another booting scenario (on LPAR) but it requires physical access to the server, which we can't do it right now as no one is sponsoring that.

Reply:
If you don't want to include the parmfile example in the code, I would create and explain them in the wiki page then.